docs/FRSKY_R9.md

```diff
@@ -21,31 +21,49 @@ The Frsky R9M transmitter module and R9MX receiver are commercially available an
   </tbody>
 </table>
 
-Comment: mLRS also supports the R9MM receiver. However, flashing the R9MM with ST-Link is really tedious and requires top soldering skills or employing other tricks, as one needs to connect to four tiny solder pads. You really should consider using the R9MX receiver instead. Note: The R9MM has a different wiring diagram than the R9MX.
+Note that R9 uses a slightly older LORA chip family, which doesn't support the 31 Hz mode. It also happens to be incompatible with the newer LORA chips when the specific spreading factor used by the mLRS 19 Hz mode is selected.  Unfortunately, this means that R9 hardware can't connect with the other mLRS boards which support 868/915 MHz.
```

Comment:
maybe "... that the R9 hardware ..."
I think too much technical explanations will only bloat text. Maybe just something along "... older LoRa chip family than boards like the Wio-E5 or E77 vased modules. Unfortunately, this means ..."

Reply:
OK on your first point.
I disagree on your second point.  Providing some clues as to the reason for this limitation may be helpful to some people who will want to know why.  It also helps explain why ELRS does not have this incompatibility in case they wondered.

Reply:
was just a suggestion, you and @jlpoltrack decide :)

Reply:
I think providing specifics is best and maybe worth adding the LoRA chipset to the table at the top.  My suggestion:  

"Note: R9 hardware uses the SX127x LoRa chipset which only supports the 19 Hz mode.  Other 868 / 915 MHz mLRS hardware (Seeed E5, eByte E77) uses the SX126x LoRa chipset.  Unfortunately, SX127x and SX126x hardware can't be used together as the spreading factor used by the 19 Hz mode (SF6) is incompatible between them.

Reply:
IMHO the table(s) (also the tables in the other pages) will be most useful if not loaded with technical schnick schnack, but what a user wants to mainly know. So I think referring to the issue in a note should be ok. 

In the note I guess I would at least wrap it upside down and first mention that there is the issue, and then give some explanations. Like currently one has to wade through lots of tec to only eventually discover the relevant statement

:)